docs/api/browser-window.md

```diff
@@ -1821,14 +1821,18 @@ there is only one tab in the current window.
 
 Adds a window as a tab on this window, after the tab for the window instance.
 
-#### `win.setVibrancy(type)` _macOS_
+#### `win.setVibrancy(type[, options])` _macOS_
 
 * `type` string | null - Can be `appearance-based`, `light`, `dark`, `titlebar`,
   `selection`, `menu`, `popover`, `sidebar`, `medium-light`, `ultra-dark`, `header`, `sheet`, `window`, `hud`, `fullscreen-ui`, `tooltip`, `content`, `under-window`, or `under-page`. See
   the [macOS documentation][vibrancy-docs] for more details.
+* `options` boolean (optional)
+  * `animate` boolean (optional) - Whether to animate or not the vibrancy toggle.
+  * `animationDuration` number (optional) - The duration of the animation in seconds.
```

Comment:
As far as I know, this is the first introduction of an API accepting a unit of time in Electron. I think it would be best to use milliseconds—as commonly used in DOM APIs—and possibly update our [best practices guidelines](https://github.com/electron/governance/blob/main/wg-api/best-practices.md) to reflect this.

Reply:
got it, refactored to milliseconds